notebooks/05_operationalize/pipeline-demo/aml-pipeline-use-datebricks-as-compute-target.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>would it be possible to provide instructions for converting the existing als deep dive notebook instead of copying that notebook and adding the .py to the repo?</p><p><br></p><p>```</p><p>jupyter nbconvert --to script notebooks/02_model/als_deep_dive.ipynb --output notebooks/05_operationalize/pipeline_demo/als_deep_dive.py</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>